### PR TITLE
release: v1.0.0b4

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -25,6 +25,11 @@
 Changes
 =======
 
+Version 1.0.0b4 (released 2017-08-09)
```
Comment (nharraud) [inline CHANGES.rst]: Your commit says `v1.0.0b3` but the here it says `1.0.0b4`

Comment (drjova) [inline CHANGES.rst]: thanks

File: CHANGES.rst [hunk 2]
```diff
@@ -25,6 +25,11 @@
...
+-------------------------------------
+
+- Fixes for Content Security Poclicy (CSP).
+
```
Comment (nharraud) [inline CHANGES.rst]: There is no need to keep the previous version here. You can take a look at other release commits [like this one](https://github.com/inveniosoftware/invenio-assets/commit/0e5fe263d359d06151854a0b93b005a031fbd59a). As long as we are in alpha/beta we just replace the previous message.

File: RELEASE-NOTES.rst [hunk 3]
```diff
@@ -1,8 +1,8 @@
-=======================
- Invenio-I18N v1.0.0b3
-=======================
```
Comment (nharraud) [inline RELEASE-NOTES.rst]: We keep the additional `=` at the end to balance the first ones. [Example](https://raw.githubusercontent.com/inveniosoftware/invenio-assets/master/RELEASE-NOTES.rst)

File: RELEASE-NOTES.rst [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
+======================
+ Invenio-I18N v1.0.0b4
+======================
 
-Invenio-I18N v1.0.0b3 was released on February 17, 2017.
+Invenio-I18N v1.0.0b4 was released on Aug 09, 2017.
```
Comment (nharraud) [inline RELEASE-NOTES.rst]: I have not seen any release note where we used abbreviated months. IMHO we should use "August" just to be consistent with the other releases.

**Pull request merged**